dace/sourcemap.py

```diff
@@ -210,12 +210,18 @@ def create_cpp_map(code: str, name: str, target_name: str):
     # We send a message to the IDE in that case to notify the user of
     # restricted features.
     if Config.get('cache') != 'hash':
+        codegen_debug = True if Config.get('compiler',
+                                           'codegen_lineinfo') else False
```

Comment:
Could Config.get('compiler','codegen_debug') be None? otherwise I can remove the if/else

Reply:
Please use `Config.get_bool`. This always ensures either true or false